2089_find_target_indices.py

```diff
@@ -0,0 +1,43 @@
+# Time complexity O(n)
+# Space complexity O(1)
+from typing import List
+
+
+class Solution:
+    
+    def targetIndices(self, nums: List[int], target: int) -> List[int]:
```

Comment:
Why did you write two kinds of binary search codes. You can do binary search much easier.